packages/hooks/useStreamQueryVariables.js

```diff
@@ -48,7 +47,12 @@ export default function useStreamQueryVariables ({
       ? (timeframe === 'future' ? 'asc' : 'desc')
       : null,
     search: null,
-    slug: !isContextGroupSlug(forGroup?.slug) ? forGroup?.slug : null,
+    // We just can't get away from some sort of jank until /all is removed from the whole repo
```

Comment:
While I agree, and believe that jank is in fact entirely contained to the getStateFromPath changes that redirect things (and the paired change to create this temporary "/my/stream" route, which is a good call, this change here is not correct or necessary. The backend knows nothing of an 'all', 'public', or 'my' slug and the previous logic was correct. This might be different due to some additional indirection on "slug" within the query maker on Web, but not here. What am I missing, and why did you make this change here?